package.json

```diff
@@ -32,6 +32,7 @@
     }
   },
   "dependencies": {
+    "@carbon/charts-react": "0.12.1",
```

Comment:
we need to remove this dependency and only have it in a peer dep I think...

Reply:
good call thanks